integration-tests/maven/src/test/java/io/quarkus/maven/CreateExtensionMojoTest.java

```diff
@@ -45,6 +45,22 @@ static CreateExtensionMojo createMojo(String testProjectName) throws IllegalArgu
         return mojo;
     }
 
+    private static Path getCopyDir(String testProjectName) {
+        Random random = new Random();
+
+        int count = 0;
+        while (count < 100) {
+            Path path = Paths.get("target/test-classes/projects/" + testProjectName + "-" + random.nextInt(1000));
```

Comment:
Just a minor question, could we have used an `UUID` instead?

Reply:
Yeah, but I don't think we should be overthinking such a small issue :)

Reply:
Done